src/handlebars/releases.handlebars

```diff
@@ -34,12 +34,78 @@
       </table>
 
       <div id="latest-downloads-container">
-        <h2 style="margin-bottom: 0.5rem;">Select a platform</h2>
-        <h3 id="latest-scroll-text" class="hide">(Scroll horizontally)</h3>
+        <h2 id="latest-select-text" style="margin-bottom: 0.5rem;">Select a platform</h2>
+        <h3 id="latest-scroll-text" class="hide">Scroll to select a platform</h3>
+        <h1 id="latest-scroll-arrows" class="animated pulse hide">
+          <i id="latest-selector-left-scroll" class="fa fa-arrow-circle-left" aria-hidden="true"></i>
```

Comment:
https://developer.mozilla.org/en/docs/Web/HTML/Element/em

Just had a read of this as I wasn't aware they were essentially the same thing...
In this case, they are not being used to make the contents italic or emphasised - purely to contain the FontAwesome arrows and other icons.

The FontAwesome site recommends using <i> tags, so I had just followed that, e.g.:
 http://fontawesome.io/icon/arrow-circle-o-right/